src/index.js

```diff
@@ -58,18 +58,21 @@ function jsxToString(component, options) {
       !Array.isArray(component.props.children)) {
       componentData.children = jsxToString(component.props.children, opts);
     } else {
-      componentData.children = component.props.children.map((child) => {
-        if (typeof child === 'string') {
-          return child;
-        } else {
-          return jsxToString(child, opts);
-        }
-      }).join(`\n${indentation}`);
+      componentData.children = component.props.children
+        .filter((child) => child)
```

Comment:
just to say on this, i though about checking for null explicitly, but after some more testing, falsey seems desirable. This is because if an empty string was used as a child (i.e. `{''}`) then we also do not want the string output to contain an empty line (i.e.  `<paren>\n      \n      </parent>`)